e/envy/envy_ubuntu_20.04.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package        : Envy
+# Version        : 1.7.1
+# Source repo    : https://github.com/gobuffalo/envy
+# Tested on      : Ubuntu 20.04 + Go latest version (v1.7.1)
```

Comment:
As per the requirement we need to test on ubi 8, is there any issue on that platform ?